SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/OperationQueueManaging/DataBrokerProtectionQueueManager.swift

```diff
@@ -250,8 +250,9 @@ private extension DefaultDataBrokerProtectionQueueManager {
     }
 
     func updateBrokerData() {
-        // Update broker files if applicable
-        brokerUpdater?.checkForUpdatesInBrokerJSONFiles()
+        Task {
+            try await brokerUpdater?.checkForUpdates()
```

Comment:
I think this should probably be blocking. As it is, brokers will be updated whilst the jobs are being executed, which feels wrong to me (and if that behavior is what we want, I think we should consider moving this responsibility outside of the queue manager)

Reply:
I'll give this some more thought. Let's follow up on this in a separate PR.